### PR TITLE
add list of regions with usernames in configuration yaml file

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

File: dashboard/config/dashboard.yml [hunk 1]
```diff
@@ -44,6 +44,7 @@ idm:
   url: ''				# IdM's authentication URL
   logoutURL: ''				# Idm's logout URL
   callbackURL: ''			# App's callback URL after auth
+  regionsAuthorized:[]      # List of regions authorized to usename: [{'RegionName': 'usernama'}]
```
Comment (flopezag) [inline dashboard/config/dashboard.yml]: username

**Comment by flopezag** [comment]
You forgive to modify the documentation in order to explain that it is necessary to define the config file for bla bla bla...

File: dashboard/README.rst [hunk 2]
```diff
@@ -204,6 +204,8 @@ rule the use of ``config/dashboard.yml`` configuration file is preferable.
 Such `file <config/dashboard.yml>`_ is self-documented, so there you will
 find a description of every configuration option.
 
+Important: If the name of your admin region username don't match with admin-<regionName>
```
Comment (flopezag) [inline dashboard/README.rst]: does not

File: dashboard/lib/routes/common.js [hunk 3]
```diff
@@ -52,7 +53,37 @@ function parseRoles(roles) {
...
+ * @param {string} regionName
+ * @param {string} username
+ */
+function checkAuthorizedByConfig(region, regionName, username) {
+    if (!region.authorized) {
+
+        var data = config.idm.regionsAuthorized;
+        data.filter(function (obj) {
+            var key = Object.keys(obj).toString();
+            var configRegionName = getRegionNameForCompare(key);
```
Comment (jframos) [inline dashboard/lib/routes/common.js:78]: When checking authorization by config, why do you need to get a substring of the username with the region name if you will have the whole username in config?

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**